docs/content/3.components/modal.md

```diff
@@ -304,11 +304,7 @@ slots:
 
 ### Programmatic usage
 
-You can use the [`useModal`](/composables/use-modal) composable to open a Modal programatically.
-
-::warning
```

Comment:
Why remove this warning as the `OverlayProvider` is needed?

Reply:
Honestly, I think we can keep it here as it's specific for the programmatic usage.